js/feed.js

```diff
@@ -0,0 +1,270 @@
+
+const username = localStorage.getItem('userName');
+const nameofuserDiv = document.querySelector('.nameofuser');
+
+// API base URL
+const API_BASE_URL = 'https://api.noroff.dev';
+
+nameofuserDiv.textContent = username;
+
+// Function to handle search when Enter key is pressed
+function handleSearchEnter(event) {
```

Comment:
There is a bug in this function. `postsData` is undefined in this context, and as a result, a user can search for a post.